src/tool_cb_hdr.c

```diff
@@ -204,7 +208,89 @@ size_t tool_header_cb(char *ptr, size_t size, size_t nmemb, void *userdata)
     if(value) {
       size_t namelen = value - ptr;
       fprintf(outs->stream, BOLD "%.*s" BOLDOFF ":", namelen, ptr);
+#ifndef LINK
       fwrite(&value[1], cb - namelen - 1, 1, outs->stream);
+#else
+      if(!strncasecmp("Location", ptr, namelen)) {
```

Comment:
I think this function call needs to be replaced with `curl_strnequal` for portability.

Reply:
Done. I can never remember which of those replacement functions is available in libcurl and which in curl.